programs/client.c

```diff
@@ -119,6 +119,7 @@ main(int argc, char *argv[])
 
 	if ((sock = usrsctp_socket(AF_INET6, SOCK_STREAM, IPPROTO_SCTP, receive_cb, NULL, 0, NULL)) == NULL) {
 		perror("usrsctp_socket");
+		return (-1);
```

Comment:
Any reason why you need to these changes as part of this PR? I don't think this is related to the single thread feature...

I would like to get this PR focussed on the single thread feature...

Reply:
ok, will revert them

Reply:
Done.